.github/workflows/ui-tests.yml

```
@@ -61,16 +61,16 @@ jobs:
         include:
           - runs-on: macos-13
             xcode: "14.3.1"
-            device: "iPhone 14 (16.4)"
+            device: iPhone 14 (16.4)
           - runs-on: macos-14
             xcode: "15.4"
-            device: "iPhone 15 (17.5)"
+            device: iPhone 15 (17.5)
           - runs-on: macos-15
             xcode: "16.4"
-            device: "iPhone 16 (18.5)"
+            device: iPhone 16 (18.5)
     with:
       fastlane_command: ui_tests_ios_swift
-      fastlane_command_extra_arguments: device:"${{matrix.device}}"
+      fastlane_command_extra_arguments: device:${{matrix.device}}
```

I'm kinda surprised that it's not something like this, but let's see how CI likes it

```suggestion
      fastlane_command_extra_arguments: "device:${{matrix.device}}"
```

---

Confirmed it works as is without the quotes